src/services/ghost/GhostServiceManager.ts

```diff
@@ -60,6 +59,7 @@ export class GhostServiceManager {
 			this.updateCostTracking.bind(this),
 			this.ghostContext,
 			this.cursorAnimation,
+			() => this.settings,
```

Comment:
avoid at least one sync by letting the inline ghost completions request latest version